Lib/warnings.py

```diff
@@ -432,9 +432,16 @@ class catch_warnings(object):
     named 'warnings' and imported under that name. This argument is only useful
     when testing the warnings module itself.
 
+    If the 'action' argument is not None, the remaining arguments are passed
+    to warnings.simplefilter() as if it that call was the first line of the
+    with-statement.
+
+    .. versionchanged:: 3.11
```

Comment:
Docstrings don't need versionchanged notes. Instead, you need to update `warnings.rst` in `Doc/` to document the new argument.

This also merits a note in whatsnew/3.11.rst.